build.rs

```diff
@@ -20,10 +21,20 @@ fn main() -> std::io::Result<()> {
     std::env::set_var("PROTOC", protoc_bin_vendored::protoc_bin_path().unwrap());
 
     if let Err(err) = get_and_build_protos(
-        &["https://raw.githubusercontent.com/cloudevents/spec/main/cloudevents/formats/cloudevents.proto", 
-        "https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/status.proto"]
+        &[
+            // cloudevent proto definitions
+            "https://raw.githubusercontent.com/cloudevents/spec/main/cloudevents/formats/cloudevents.proto", 
+
+            // uProtocol-project proto definitions
+            "https://raw.githubusercontent.com/eclipse-uprotocol/uprotocol-core-api/main/src/main/proto/uuid.proto",
```

Comment:
in order to prevent these from becoming a *moving target* I would propose to pin a particular version (1.5.3?) instead of using whatever is currently on `main` ...

Reply:
done